src/components/ezp-printer-selection/ezp-printer-selection.tsx

```diff
@@ -381,6 +388,16 @@ export class EzpPrinterSelection {
         this.selectedProperties.paperwidth = eventDetails.value
         // console.log(this.selectedProperties)
         break
+      case 'tray':
+        if(this.selectedPrinterConfig.Trays && this.selectedPrinterConfig.Trays.length >= 0 && this.selectedPrinterConfig.Trays[0] == null) {
```

Comment:
Why do we have this condition here? Can't we just keep it in one place right before the payload is send to the API?